luaui/RmlWidgets/gui_awards/gui_awards.rcss

```diff
@@ -0,0 +1,171 @@
+* {
+    font-family: "Exo 2";
+    font-size: 18px;
+    color: white;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  background-color: rgba(255,255,255,0.7)
```

Comment:
the mix of rgba() and hexadecimal is no good. Stick with just hexadecimal which does still have an alpha component. I think you will find though that this property isn't even working in your implementation.